hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/protocol/EndpointService.java

```diff
@@ -282,7 +293,13 @@ public void measureMetrics(Collection<MeasurementInstance<L, MetricType<L>>> ins
             Driver<L> driver = session.getDriver();
             for (MeasurementInstance<L, MetricType<L>> instance : instances) {
                 AttributeLocation<L> location = instance.getAttributeLocation();
-                Object o = driver.fetchAttribute(location);
+                Object o = null;
+                try (Context timerContext = diagnostics.getRequestTimer().time()) {
+                    o = driver.fetchAttribute(location);
+                } catch (Exception e) {
+                    diagnostics.getErrorRate().mark(1);
```

Comment:
and this is wrong - if we can an exception here, this isn't necessarily from a problem with the DMR connection - it could have been an exception from else where. Again, the error rate for the DMR diagnostics is ONLY to capture errors we had in the DMR communications, nothing extra inside of the agent should trigger a DMR error rate increment.